src/ngraph/frontend/onnx_import/op/supported_ops.md

```diff
@@ -0,0 +1,137 @@
+## Supported Ops:
+
+Generally nGraph doesn't support tensors of types:
+
+* `string`,
+* `float16`,
+* `complex64`,
+* `complex128`.
+
+Value in `()` _brackets_ means that this op is available since specific opset version.
+Values seperated by `-` _dash_ represents opset versions introducing some changes. 
+In example operator `Abs` is supported in all opset versions starting from `1` to `6` 
```

Comment:
For example, with the schema represented below, the operator `Abs` is supported in ...